101-vm-with-data-management-gateway/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "create a data management gateway and install in azure VM",
+  "description": "This template deploys a virtual machine and creates a workable data management gateway",
+  "summary": "Install and register gateway on an windows virtual machine using custom script extension",
```

Comment:
typos
Install and register a gateway on a Windows virtual machine using the customer script extension